### PR TITLE
st/utils: fix err handling & coding style issues

### PR DESCRIPTION
**Pull request by andriytk**
Fix errors handling and coding style issues.

### CODE REVIEW

**Comment by cortx-admin** [comment]
<h1>Jenkins CI Result : <a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411/>Motr#411</a></h1><h2>Motr Test Summary</h2><table><tr><th>Test Result</th><th>Count</th><th>Info</tr><tr><td>:x:Failed</td><td>1</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/49motr-rpc-cancel target=_blank>04motr-single-node/49motr-rpc-cancel</a></p></details></td></tr><tr><td>:checkered_flag:Skipped</td><td>5</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/37protocol target=_blank>01motr-single-node/37protocol</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/31motr-sys-kvs-kernel-test target=_blank>04motr-single-node/31motr-sys-kvs-kernel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/07multi-client-tests target=_blank>05motr-single-node/07multi-client-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/39mount-fail target=_blank>05motr-single-node/39mount-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/12sns-cc-io-tests target=_blank>05motr-single-node/12sns-cc-io-tests</a></p></details></td></tr><tr><td>:heavy_check_mark:Passed</td><td>58</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/46m0crate target=_blank>01motr-single-node/46m0crate</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/01kernel-tests target=_blank>01motr-single-node/01kernel-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/00userspace-tests target=_blank>01motr-single-node/00userspace-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/03initscript-tests target=_blank>01motr-single-node/03initscript-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/30hagen-test target=_blank>01motr-single-node/30hagen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/29confgen-test target=_blank>01motr-single-node/29confgen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/35m0singlenode target=_blank>01motr-single-node/35m0singlenode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/04net-tests target=_blank>01motr-single-node/04net-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/05rpcping-tests target=_blank>01motr-single-node/05rpcping-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/28motr-sys-kvs-test target=_blank>01motr-single-node/28motr-sys-kvs-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/06console-tests target=_blank>01motr-single-node/06console-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/01motr-single-node/52singlenode-sanity target=_blank>01motr-single-node/52singlenode-sanity</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/20pool-version-test target=_blank>02motr-single-node/20pool-version-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/21spiel-test target=_blank>02motr-single-node/21spiel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/47motr-tests-user-kernel target=_blank>02motr-single-node/47motr-tests-user-kernel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/09poolmach-tests target=_blank>02motr-single-node/09poolmach-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/41motr-conf-update target=_blank>02motr-single-node/41motr-conf-update</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/08degraded-mode-tests target=_blank>02motr-single-node/08degraded-mode-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/24m0d-fsync-test target=_blank>02motr-single-node/24m0d-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/02system-tests target=_blank>02motr-single-node/02system-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/22rpc-cancel-test target=_blank>02motr-single-node/22rpc-cancel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/53clusterusage-alert target=_blank>02motr-single-node/53clusterusage-alert</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/23m0d-signal-test target=_blank>02motr-single-node/23m0d-signal-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/02motr-single-node/51kem target=_blank>02motr-single-node/51kem</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/54sns-repair-motr-1f target=_blank>03motr-single-node/54sns-repair-motr-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/59sns-repair-motr-abort target=_blank>03motr-single-node/59sns-repair-motr-abort</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/17sss-st-test target=_blank>03motr-single-node/17sss-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/58sns-repair-motr-ios-fail target=_blank>03motr-single-node/58sns-repair-motr-ios-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/60sns-repair-motr-abort-quiesce target=_blank>03motr-single-node/60sns-repair-motr-abort-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/55sns-repair-motr-1n-1f target=_blank>03motr-single-node/55sns-repair-motr-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/56sns-repair-motr-mf target=_blank>03motr-single-node/56sns-repair-motr-mf</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/61spiel-multi-confd target=_blank>03motr-single-node/61spiel-multi-confd</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/16conf-st-test target=_blank>03motr-single-node/16conf-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/57sns-repair-motr-1k-1f target=_blank>03motr-single-node/57sns-repair-motr-1k-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/03motr-single-node/36spare-reservation target=_blank>03motr-single-node/36spare-reservation</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/51motr-rmw target=_blank>04motr-single-node/51motr-rmw</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/50motr-rm-lock-cc-io target=_blank>04motr-single-node/50motr-rm-lock-cc-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/25spiel-sns-repair-test target=_blank>04motr-single-node/25spiel-sns-repair-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/33motr-st target=_blank>04motr-single-node/33motr-st</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/19sns-abort-test target=_blank>04motr-single-node/19sns-abort-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/32m0t1fs-rconfc-fail-test target=_blank>04motr-single-node/32m0t1fs-rconfc-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/34sns-repair-1n-1f target=_blank>04motr-single-node/34sns-repair-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/26spiel-sns-repair-quiesce-test target=_blank>04motr-single-node/26spiel-sns-repair-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/27sns-repair-io-fail-test target=_blank>04motr-single-node/27sns-repair-io-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/04motr-single-node/48motr-raid0-io target=_blank>04motr-single-node/48motr-raid0-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/40motr-dgmode target=_blank>05motr-single-node/40motr-dgmode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/42motr-utils target=_blank>05motr-single-node/42motr-utils</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/13sns-repair-quiesce target=_blank>05motr-single-node/13sns-repair-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/45motr-sns-repair-N-1 target=_blank>05motr-single-node/45motr-sns-repair-N-1</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/11sns-multi-tests target=_blank>05motr-single-node/11sns-multi-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/44motr-sns-repair target=_blank>05motr-single-node/44motr-sns-repair</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/14m0t1fs-fsync-test target=_blank>05motr-single-node/14m0t1fs-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/43motr-sync-replication target=_blank>05motr-single-node/43motr-sync-replication</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/38sns-abort-quiesce-test target=_blank>05motr-single-node/38sns-abort-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/15m0t1fs-fwait-test target=_blank>05motr-single-node/15m0t1fs-fwait-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/18m0mount-test target=_blank>05motr-single-node/18m0mount-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/05motr-single-node/10sns-single-tests target=_blank>05motr-single-node/10sns-single-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit/motr-single-node/62dix-repair-st-lookup-insert target=_blank>motr-single-node/62dix-repair-st-lookup-insert</a></p></details></td></tr><tr><td>Total</td><td>64</td><td><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/411//testReport/junit>:link:</a></td></tr></table><h2> CppCheck Summary</h2><p> &emsp;&emsp;&emsp;Cppcheck: No new warnings found :thumbsup:</p>

File: motr/st/utils/helper.c [hunk 1]
```diff
@@ -219,19 +218,16 @@ static int open_entity(struct m0_entity *entity)
...
+		return M0_ERR(rc);
 
 	m0_op_launch(ops, 1);
 	rc = m0_op_wait(ops[0], M0_BITS(M0_OS_FAILED,
-					       M0_OS_STABLE),
-			       M0_TIME_NEVER);
+					M0_OS_STABLE), M0_TIME_NEVER);
 	if (rc == 0)
-		rc = ops[0]->op_rc;
+		rc = m0_rc(ops[0]);
```
Comment (huanghua78) [inline motr/st/utils/helper.c:227]: Personally, I am thinking the function name should be changed to m0_op_rc().

File: motr/st/utils/helper.c [hunk 2]
```diff
@@ -285,38 +275,22 @@ static int write_data_to_object(struct m0_obj *obj,
...
-	int                  nr_tries = 10;
 	struct m0_op        *ops[1] = {NULL};
 
-again:
-
-	/* Create the write request */
-	m0_obj_op(obj, M0_OC_WRITE, ext, data, attr, 0, 0, &ops[0]);
-	if (ops[0] == NULL)
-		return M0_ERR(-EINVAL);
+	/* Create write operation */
```
Comment (huanghua78) [inline motr/st/utils/helper.c:280]: I don't remember the reason why we re-try to launch the write operation.
And I am not sure if that problem has been fixed.

File: motr/st/utils/helper.c (hunk 1 above)
Comment (andriytk) [reply motr/st/utils/helper.c:227]: I agree. It can be done in another PR.

File: motr/st/utils/helper.c (hunk 2 above)
Comment (andriytk) [reply motr/st/utils/helper.c:280]: Exactly. So better to clean up the rubbish after ourselves. Especially, as we present this code as a reference one to the community.

File: motr/st/utils/copy.c [hunk 3]
```diff
@@ -116,21 +116,19 @@ int main(int argc, char **argv)
 		      cp_param.cup_update_mode);
 	if (rc < 0) {
 		if (rc == -EEXIST) {
-			fprintf(stderr, "Object id: " U128X_F "exists. "
-					"To update an existing object, "
-					"use -u=start index! rc = %d\n",
-					 U128_P(&cp_param.cup_id),
-					 rc);
+			fprintf(stderr, "Object "U128X_F" already exists: "
+				"rc=%d. To update an existing object, use"
```
Comment (huanghua78) [inline motr/st/utils/copy.c:120]: I myself prefer to old code.

String on this line, is continuous to the one on the previous line.
So, they are aligned.

File: motr/st/utils/helper.c [hunk 4]
```diff
@@ -735,24 +674,26 @@ int m0_unlink(struct m0_container *container,
...
 
 	m0_entity_delete(&obj.ob_entity, &ops[0]);
+
 	m0_op_launch(ops, 1);
-	rc = m0_op_wait(ops[0],
-			       M0_BITS(M0_OS_FAILED,
-				       M0_OS_STABLE),
-			       M0_TIME_NEVER);
+	rc = m0_op_wait(ops[0], M0_BITS(M0_OS_FAILED,
+					M0_OS_STABLE), M0_TIME_NEVER);
```
Comment (huanghua78) [inline motr/st/utils/helper.c:693]: Just like this one:
M0_OS_STABLE is aligned with M0_OS_FAILED.

File: motr/st/utils/copy.c (hunk 3 above)
Comment (andriytk) [reply motr/st/utils/copy.c]: Ready, please check.

**Comment by huanghua78** [comment]
Let's ask @siningwuseagate to review this PR.

**Review by siningwuseagate — APPROVED** [review]

**Comment by andriytk** [comment]
Interesting: during the `49motr-rpc-cancel` test sometimes there is `Cancelled session` error from m0cp/m0cat utilities and sometimes not. Compare:

```
$ cat /var/motr/systest-37999/motr/m0cp.log 
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e640  ERROR  [rpc/session.c:578:m0_rpc_session_validate]  <! rc=-125 Cancelled session
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  e6f0  ERROR  [rpc/rpc.c:126:m0_rpc__post_locked]  <! rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f14600afd00 rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:210:io_bottom_half]  ioo=0x1da9880 from=192.168.180.171@tcp:12345:33:901 rc=-125 ti_rc=-125 @<4300000600000000:100004>
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f14600b0800 rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f14600abb00 rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f14600ac610 rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f146009aec0 rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f146009b960 rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f146009ed30 rc=-125
motr[39265]:  ea80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x1da9880] rpc item 0x7f146009f510 rc=-125
motr[39265]:  e8e0  ERROR  [io_req.c:1341:device_check]  <! rc=-5 [0x1da9880] Failed to recover data since number of failed data units (0) exceeds number of parity units in parity group (2) OR number of failed services (4) exceeds number of max failures supported (0)
motr[39265]:  eab0  ERROR  [io_req.c:659:ioreq_iosm_handle_executed]  iro_dgmode_write() failed: rc=-5
Writing to object failed!
```

vs.:

```
$ cat /var/motr/systest-36385/motr/m0cp.log 
motr[37656]:  6a80  ERROR  [io_req_fop.c:161:io_bottom_half]  [0x221c010] rpc item 0x7fd52804c9e0 rc=-125
motr[37656]:  6a80  ERROR  [io_req_fop.c:210:io_bottom_half]  ioo=0x221c010 from=192.168.180.171@tcp:12345:33:901 rc=-125 ti_rc=-125 @<4300000700000000:100004>
motr[37656]:  6ab0  ERROR  [io_req.c:615:ioreq_iosm_handle_executed]  ioo->ioo_rc=-125
Writing to object failed!
```

I don't see how the changes in this PR could affect this.

**Comment by andriytk** [comment]
It's because of the wrong old errors handing at `write_data_to_object()` and `read_data_from_object()` when we didn't check `rc` from `m0_op_wait()`. So the functions would endlessly try to do ops even when the session is cancelled, so there were more chances for such error to appear in the log.

I think the test should check for `rc=-125` error instead of `Cancelled session` msg. It's safer.

**Comment by andriytk** [comment]
The fix - https://github.com/Seagate/cortx-motr/pull/698

**Pull request merged**